lightwood/mixers/nn/nn.py

```diff
@@ -83,7 +86,9 @@ def build_confidence_normalization_data(self, ds, subset_id=None):
 
             with torch.no_grad():
                 if self.is_selfaware:
-                    outputs, awareness = self.net(inputs)
+                    self.selfaware_net.eval()
```

Comment:
it seems like the main network is not set to `eval`, if so we should also do this here. If it is set, but somewhere else, we should set the selfaware to eval there as well.

Reply:
It is set at the start of this function, so as it stands it seems fine to call .eval() for the self-aware here.